Helpers.fs

```diff
@@ -40,18 +57,32 @@ module Promise =
     let either a  (b: Func<obj, U2<'R, PromiseLike<'R>>>) (pr : Promise<'T>) : Promise<'R> =
```

Comment:
Can this `Func` be shifted into the body of `either` so that it takes a normal F# function as its arg instead?